packages/node_modules/@webex/widget-space/src/container.js

```diff
@@ -148,10 +145,7 @@ export class SpaceWidget extends Component {
       const {preferredWebexSite} = sparkInstance.meetings;
       let isMeetButtonDisabled = false;
 
-      if (sparkInstance.meetings.getMeetingByType(CONSTANTS.CONVERSATION_URL, currentConversation)) {
-        isMeetButtonDisabled = false;
-      }
-      else {
+      if (props.destinationType === 'spaceId' && activityTypes.some((activity) => activity.name === 'meet')) {
```

Comment:
How is the flow? if user trying to join with CONVERSATION_URL ??
Earlier we were hiding the meet button based on the conditions, those configuration changes were for migrating to V2 meetings.

Reply:
In space-widgets, I don't see the option of joining with CONVERSATION_URL. Could you please help me with steps for the same?
The existing condition was blocking flow for SIP URI but not for SpaceID so I updated the condition here to reverse that.

Reply:
It's the same as the way we have in the KS samples sdk app, in KS app also, join button disabled, 

<img width="1210" alt="image" src="https://github.com/webex/react-widgets/assets/3918217/59cc06ed-eb38-4667-be0e-d82d25ebcbb3">

Reply:
In the kitchen sink app, the dropdown has a conversation URL as one of the options, there is nothing similar in space widgets. I only see 4 options to be chosen as destination type: To Space, To Person, To PersonId, To SIP URI. Now which one of these would be the case for conversation URL?

<img width="268" alt="Screenshot 2024-06-11 at 5 13 34 PM" src="https://github.com/webex/react-widgets/assets/65543166/0cd4973f-8452-442a-8ae9-0ca6ab504301">

Reply:
oh yes, we don't have convURL here, You can choose any options and try passing the convURL and check,? In SDK there's utility meeting-info which figure out based on input what type of destination it is. Just to test once..

Reply:
I have tried choosing the 'To Space' option and entered the conversation URL and it says unable to load space.